hhast-lint.json

```diff
@@ -6,7 +6,6 @@
     {
       "patterns": ["*"],
       "disabledLinters": [
-        "Facebook\\HHAST\\Linters\\MustUseOverrideAttributeLinter",
```

Comment:
🤔

Reply:
We don't need to disable linters, because we lint clean.
I can actually also remove the the DontUseReferences exception.

Reply:
Gotcha, nice.